pootle/core/utils/stats.py

```diff
@@ -26,3 +29,24 @@ def make_dict(state, title, filter_url=True):
     states.append(make_dict('untranslated', _("Untranslated")))
 
     return states
+
+
+def get_top_scorers_data(top_scorers, chunk_size):
+    all_scorers_loaded = len(top_scorers) <= chunk_size
+
+    top_scorers_data = [
+        dict(
```

Comment:
If I am not wrong this is what @julen suggests:

```python
    top_scorers_data = [
        {
            'public_total_score': scorer['public_total_score'],
            'suggested': scorer['suggested'],
            'translated': scorer['translated'],
            'reviewed': scorer['reviewed'],
            'email': scorer['user'].email,
            'display_name': scorer['user'].display_name,
            'username': scorer['user'].username,
        } for scorer in top_scorers[:chunk_size]
    ]
```

but despite I find this a bit clearer I don't think it is worth doing the change.

@julen `[{'foo': foo.bar} for foo in baz]` is not dict comprehension (it is list comprehension) while `{x: x**2 for x in (2, 4, 6)}` is.